### ISSUE TITLE
Is it supposed to work with gettext?

### ISSUE THREAD

**Issue by michelem09** [andrewrk/swig-email-templates#24]
My email template contains gettext from i18n-abide module but I cannot get it working. The result string disappears in the translated email. (of course same string works in a template web view, not email)

Template ha strings like this:

```<p><%= gettext('Welcome to') %></p>```

**Comment by takkaria** [comment]
Hi, is this still an issue you're having?

**Comment by michelem09** [comment]
oh well, I should try because I solved this with a workaround writing two templates one for EN locale and one for IT.
Let me try again with the 2.0 version I'm going to install.

**Comment by burakkilic** [comment]
Hi @michelem09 , did you solve that? I also need to extract and translate with gettext.

**Comment by takkaria** [comment]
I would be happy to spend some time trying to make this work if I was given an example of something that should work but didn't.

**Comment by burakkilic** [comment]
Let's say we have annotation like
    
    <h4>{% trans "You have a new feedback." %}</h4>

I can extract it with jsxgettext. But I don't know how to compile and use the translations in the template.

**Comment by takkaria** [comment]
Ah, so this isn't specifically a problem with email templates, this is a problem generally with integrating gettext and swig?

**Comment by burakkilic** [comment]
Yes, I think so.

**Comment by halt-hammerzeit** [comment]
I can possible answer this issue if any of you explains in proper English what issue are you having.
@burakkilic @michelem09 

As for me, I made it working with my own simple implementation of translation.
It is really no more than making a `JSON` translation and then doing `return JSON[key]`.
No `gettext` or anything is needed.

**Comment by takkaria** [comment]
I'd also suggest switching to [email-templates](https://github.com/crocodilejs/node-email-templates) if you want built-in locale support.  It doesn't use gettext but has separate files for each locale.  Each file could inherit from a base template and have some macros so there is only one place to change e.g. URLs.

**Comment by halt-hammerzeit** [comment]
(I disagree with the idea of separate files for separate locales because of artificial duplication, but they might have their own opinion on that matter.)
